bundles/EcommerceFrameworkBundle/src/Tracking/Tracker/Analytics/AbstractAnalyticsTracker.php

```diff
@@ -33,7 +34,7 @@ abstract class AbstractAnalyticsTracker extends EcommerceTracker
     public function __construct(
         TrackingItemBuilderInterface $trackingItemBuilder,
         Environment $twig,
-        Tracker $tracker,
+        TrackerInterface $tracker,
```

Comment:
we should add `GoogleMarketingBundle` as dependent bundle to `EcommerceFrameworkBundle`